MAUI/TextInputLayout/Fixed-Hint-Position.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: Fixed Hint Position in .NET MAUI TextInputLayout control | Syncfusion
+description: Learn here all about Fixed Hint Position support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Fixed Hint Position in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Hint label for the text input layout is fixed always at the top position. This helps users make the hint label floating even when the input view is not focused. It can be enabled by setting the `IsHintAlwaysFloated` property.
```

Comment:
Hint label for the text input layout is always fixed at the top position. This helps users make the hint label float even when the input view is not focused. It can be enabled by setting the `IsHintAlwaysFloated` property.

Reply:
Addressed the changes